packages/rancher-gatekeeper/generated-changes/exclude/templates/gatekeeper-system-namespace.yaml

```diff
@@ -1,14 +0,0 @@
-{{- if .Values.createNamespace }}
```

Comment:
seems like this no longer exists in chart. LGTM

Reply:
This file is removed from upstream v3.4.0 onwards